### PR TITLE
Thorn eel campaign tweaks

### PR DESCRIPTION
**Pull request by Thorneel**
In some missions, replaced the objective buildings, from Artefact to more appropriate ones such as Relays or Wormhole Generators

In the Codex, tweaked the entry of New Falsell so it works slightly better, plus added some comments on some of the entries (and removed some that were moved to the lore file)

In the Lore file, replaced the comments by a more detailed text:
- An overview of the place of Commanders in a war, among other things explaining why the typical multiplayer game is actually a typical Commander engagement, but also not the only type of engagement there is
- A brief history of the galaxy, with some general comments on the Planetwars era, a rethreading of the events of the campaign with a few extra details, and some general comments on the Multiplayer era as the age of stars draws to an end
- A few comments on the writing of the campaign, inspirations and writing decisions

Feedback welcome on the lore document, if there are inconsistencies or things that don't work well to be fixed notably

### CODE REVIEW

**Comment by GoogleFrog** [comment]
Have you checked that the alternate buildings physically fit in the mission? Also I recall Artefacts being invulnerable while other structures are not, so ideally the new structures would not be anywhere dangerous,

File: campaign/sample/lore.txt [hunk 1]
```diff
@@ -1,17 +1,393 @@
...
+
+Commander insertion can only be done with either an already established foothold on the target world, or with heavy warship support from space, or with a foothold on a nearby celestial body if one exists that is within teleportation range.
+Attacking a world without having local supporters already establishing a foothold is difficult and requires considerable orbital forces, but concentration of forces can allow great powers to overwhelm individual planetary defenses relatively easily. In theory at least.
+
+Note that outside extreme concentration of space forces, the anti-space defenses of a typical planet can easily keep attackers at bay and prevent them from direct assault. And even with extreme concentration of force, such assault would be extremely costly.
+Hence why inserting Commanders to build bridgeheads is needed instead.
+
+It is easier to neutralize long-range fires than to launch them, thus attackers and defenders generally neutralize each-other�s fires and the Commanders on the surface don�t have fire support they don�t build themselves.
```
Comment (sprunk) [inline campaign/sample/lore.txt:16]: something is going on here with the `'` character

Comment (Thorneel) [reply campaign/sample/lore.txt]: Oops, fixed

**Comment by Thorneel** [comment]
> Have you checked that the alternate buildings physically fit in the mission? Also I recall Artefacts being invulnerable while other structures are not, so ideally the new structures would not be anywhere dangerous,

I have launched the missions to check and there doesn't seem to be a size issue. The footprint seems similar to the one of the Artifact building in both cases.
In the mission file, they are marked as `invincible` and `ignoredByAI`. I have tested invincibility by mass-bombing them with cheated Krows, with no effect on their health.

**Comment by GoogleFrog** [comment]
Right. I don't know whether it is worth caring, but we may have decided that artefacts are invincible while other structures can be killed. This would be a sort of consistency that is only enforced via configuration. It could be checked whether this was the case.

**Comment by Thorneel** [comment]
There are other special buildings from Planetwars in the campaign that are invincible. The pw_warpgate and pw_warpjammer are used as invincible objective buildings in some missions, and some missions have invincible decorative buildings like in planet7 (New Falsell) which has half a dozen different ones.

In the latter, we can see buildings like pw_metal are invincible in some missions, as decorative neutral units, but not others where they are active units given to a side at the start and can be destroyed. I feel like it works when the decorative are out of the way so to speak and won't be confused with active units that need to be destroyed (such as New Falsell where static defenses only don't make it look like it is a part of an active base). But if that doesn't feel right, those could be removed or replaced by other decorative buildings that would be decorative buildings in all missions.

The objective or decoration buildings being instead destructible targets in a future Planetwars game is, I think, not an issue as it is a very different context, and the rules and objectives in Planetwars would be clear about those.

File: campaign/sample/lore.txt [hunk 2]
```diff
@@ -1,17 +1,435 @@
...
+
+
+Silence Era
+
+- After the Event, some of the few survivors in deep hibernation started to wake up in an emptied galaxy. Most infrastructure and automated systems were still running, and while some areas still had active military bots and other automated hazards, most of the galaxy was mostly safe.
+Some of the survivors managed to get interstellar communication and starships working, contacting each-other and searching worlds for other survivors. They soon discovered the final evacuation plans of the Empire towards the Eternity Gate. They decided to follow them, as many evacuation ships were still present and functional, and they judged they were too few to really restart civilization in this galaxy.
+Knowing that other survivors may wake up after themselves, as there were little chances that they had found everyone, they prepared things for others trying to follow them afterwards.
+They left message beacons running, put supplies together in many places, deactivated some dangerous automated systems, set up warp gates for long-term autonomous function and self-repair, left control codes they could find for still active military automata�
```
Comment (sprunk) [inline campaign/sample/lore.txt:330]: This sentence also has some weird character at the end, is it a broken period or was the sentence cut off?
image

File: campaign/sample/lore.txt [hunk 3]
```diff
@@ -1,17 +1,435 @@
...
+
+You will note that MC, beyond what we learn of the Demiurge project, is left entirely nondescript. This is by design.
+First, this is from MC's point of view, with a narrative first person, and there is no-one else to interact with, so what we see is entirely outwards by nature.
+Yes, there is no other character because it would have been difficult to make work in the structure of the campaign, which was complete before the writing was started. I'll just say that limitations breed art and leave it at that�
+Secondly, and this is an old trick in videogames for good reasons, this lets the player project themselves on MC, which helps immersion. You are not telling a character what to do next, you are doing the things the character is doing, discovering what they discover, and so on.
+
+Regarding the New Falsell anomaly, well, the first time I played that mission at the time, I had that eerie feeling: all those heavy defences, and yet it felt so empty� So I took inspiration from that first impression.
```
Comment (sprunk) [inline campaign/sample/lore.txt:423]: More weird characters.
image

File: campaign/sample/lore.txt [hunk 4]
```diff
@@ -1,17 +1,435 @@
...
+
+- Old Earth is initially dominating the panhuman sphere, but slowly looses its hegemonic position to new centers of power, as the wider galaxy is settled
+
+- Old Earth becomes a minor power, then a historical sanctuary, before being slowly forgotten. Some believe that the Dynasty used it as headquarters for millennia and were the ones responsible for its gradual oblivion. However, even they seem to have moved out at some point and forgotten about Earth themselves
+
+- Technological progress gradually plateaus in most fields, with new advances mostly rediscovering forgotten older ones
+
+- The first known recorded Chicken sighting is somewhere around here, though it seems that they were already a known if rare threat at the time.
+
+- The galaxy alternates between periods of peace, war, prosperity, crises� As the old saying goes, whose origins have long been lost to time: united they shall divide, divided they shall unite, thus it has always been, thus it will always be
```
Comment (sprunk) [inline campaign/sample/lore.txt:103]: image

File: campaign/sample/lore.txt [hunk 5]
```diff
@@ -1,17 +1,435 @@
...
+Similarly, there is no need to force oneself to adhere to strict realism beyond what is needed for intuition to hold on how the world generally works, as long as things are interesting and consistent.
+
+Now this is not to say that there are no contrivances at all. Some contrivances can be good, as long as they don't feel too forced. And sometimes, explaining some odd bit can actually lead to interesting places. Or, conversely, pointing it out as an unexplained oddity, as something that people realize is strange and hinting at a mystery.
+On the other hand, sometimes ignoring minor inconsistencies entirely can also work, as long as they generally would not be noticed. If the rest works well enough, the player can accept small details not quite lining up, or even assume that there are explanations that are just not stated, just like everything else appears consistent enough.
+
+This was the philosophy used to write this lore, hopefully it didn't fail too badly at it�
```
Comment (sprunk) [inline campaign/sample/lore.txt:26]: More weird characters here and above.

**Comment by Thorneel** [comment]
Ok I've found what the issue was with those � characters
I've been using libreoffice for writing this, and it automatically replaces certain characters by others, notably ... by … and " with “ and ”
They should all be gone now
Also I am using ... way too much but hopefully it won't be too much of an issue as long as it's confined in the lore document

File: campaign/sample/lore.txt (hunk 2 above)
Comment (Thorneel) [reply campaign/sample/lore.txt]: Fixed

File: campaign/sample/lore.txt (hunk 3 above)
Comment (Thorneel) [reply campaign/sample/lore.txt]: Fixed

File: campaign/sample/lore.txt (hunk 4 above)
Comment (Thorneel) [reply campaign/sample/lore.txt]: Fixed

File: campaign/sample/lore.txt (hunk 2 above)
Comment (Thorneel) [reply campaign/sample/lore.txt]: Fixed

File: campaign/sample/lore.txt (hunk 5 above)
Comment (Thorneel) [reply campaign/sample/lore.txt]: Fixed

**Review by sprunk — APPROVED** [review]
LGTM

**Pull request merged**